consular/cli.py

```diff
@@ -59,17 +60,16 @@ def main(scheme, host, port,
 
     log.startLogging(logfile)
 
-    consular = Consular(consul, marathon, fallback)
+    consular = Consular(consul, marathon, fallback, registration_id)
     consular.debug = debug
     consular.timeout = timeout
     consular.fallback_timeout = fallback_timeout
-    if registration_id:
-        events_url = "%s://%s:%s/events?%s" % (
-            scheme, host, port,
-            urlencode({
-                'registration': registration_id,
-            }))
-        consular.register_marathon_event_callback(events_url)
+    events_url = "%s://%s:%s/events?%s" % (
+        scheme, host, port,
+        urlencode({
+            'registration': registration_id,
+        }))
+    consular.register_marathon_event_callback(events_url)
```

Comment:
I didn't do this earlier (sorry!) but it probably makes sense to add an `errback` to this one to make sure errors are handled somewhat ok (say if we try and start consular without marathon being available).

Also there's a potential race condition here because we're registering the callback before consular itself is running.